### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by mook-as**
This adds the acceptance tests, and required packages.

Most of the functionality regarding configuration isn't tested yet; I've only confirmed that CATS can _start_.

### CODE REVIEW

**Comment by mook-as** [comment]
Group configuration is done; we can now limit to specific test groups.  Should be ready for review.

Not sure if the package vendoring via bosh works; might need to copy the sources instead…

File: jobs/acceptance-tests/templates/run.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -e
+
+<% if p("bpm.enabled") %>
+exec /var/vcap/jobs/bpm/bin/bpm run smoke_tests
```
Comment (f0rmiga) [inline jobs/acceptance-tests/templates/run.erb]: This is wrong.

File: jobs/acceptance-tests/templates/config.json.erb [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<%
+require 'json'
+
+config = p('acceptance_tests')
+config[:use_existing_user] = !properties.acceptance_tests.existing_user.nil?
+config[:keep_user_at_suite_end] = !properties.acceptance_tests.existing_user.nil?
+config[:use_existing_organization] = !properties.acceptance_tests.existing_organization.nil?
+config[:artifacts_directory] ||= '/var/vcap/sys/log/acceptance_tests/'
+
+# TODO: do something with p('acceptance_tests.include')
```
Comment (f0rmiga) [inline jobs/acceptance-tests/templates/config.json.erb]: Is this TODO still relevant? We are doing _something_ with it below.

**Review by f0rmiga — CHANGES_REQUESTED** [review]

File: jobs/acceptance-tests/templates/config.json.erb (hunk 2 above)
Comment (mook-as) [reply jobs/acceptance-tests/templates/config.json.erb]: Fixed (comment removed) in 294f869e844409006b8de0f79291e1e904ecb3ed

File: jobs/acceptance-tests/templates/run.erb (hunk 1 above)
Comment (mook-as) [reply jobs/acceptance-tests/templates/run.erb]: Fixed in 294f869e844409006b8de0f79291e1e904ecb3ed ; though I notice that things still magically worked, _somehow_, even though `bpm.enabled` is true…

**Review by f0rmiga — APPROVED** [review]

**Pull request merged**